Yank/repex.py

```diff
@@ -754,11 +769,11 @@ def read_mixing_statistics(self, iteration=slice(None)):
 
         Returns
         -------
-        n_accepted_matrix : kxk numpy.ndarray
+        n_accepted_matrix : numpy.ndarray with shape (n_states, n_states)
```

Comment:
Is this going to be left for Repex only (`n_states x n_states` swapping)? That is, will the `SAMS` reporter have a different Mixing statistics output, or just a different way its calculated?

Reply:
No, we can still build the empirical transition matrix on `n_states x n_states` for SAMS.